quickwit-actors/src/actor.rs

```diff
@@ -362,6 +382,7 @@ pub(crate) fn process_command<A: Actor>(
             ctx.pause();
             None
         }
+        Command::Success => Some(ActorExitStatus::Success),
```

Comment:
rename to Command::Finish

Reply:
I see how `Command::Success` is bothering you. `EndWithSuccess`, `ExitWithSuccess`, `FinishWithSuccess`, `QuitWithSuccess`, `TerminateWithSuccess`...? :D

Reply:
Went for ExitWithSuccess